features/system-color.yml

```
@@ -0,0 +1,10 @@
+name: system-color
+description: The `system-color` CSS keywords allow authors to match default colors from the user agent.
```

As written, this suggests `system-color` is a keyword you can actually type and get something from. I think that's not the case. I'd suggest something along these lines, using examples to represent the real-world use:

```suggestion
description: The system color CSS keywords, such as `ButtonBorder` or `LinkText`,  allow authors to match default colors from the user agent.
```